.gitignore

```diff
@@ -0,0 +1,38 @@
+# OS X Finder
+.DS_Store
+._*
+.Spotlight-V100
+.Trashes
+
+# Windows
+Thumbs.db
+ehthumbs.db
+Desktop.ini
+$RECYCLE.BIN/
+
+lib-cov
+*.seed
+*.log
+*.csv
+*.dat
+*.out
+*.pid
+*.gz
+
+pids
+logs
+results
+
+node_modules
+
+# IntelliJ
+.idea
+*.iml
+
+coverage
+
+/config/runtime.json
+
+# Ignore build output
+lib
+.eslintcache
```

Comment:
- [ ] You need a `.npmignore` file with the same content except that it ignores src and test